docs/user/power/install_upi.md

```diff
@@ -0,0 +1,296 @@
+# Install: Power User Provided Infrastructure
+
+The steps for performing a UPI-based install are outlined here. Example automation is provided [here](#example-power-upi-configuration) to help model your own.
+
+## Table of contents
+
+1. [Minimum compute requirements](#minimum-resource-requirements)
+
+2. [Network topology requirements](#network-topology-requirements)
+
+3. [DNS requirements](#dns-requirements)
+
+4. [Getting Ignition configs for machines](#getting-ignition-configs-for-machines)
+
+5. [Booting machines with RHCOS and Ignition configs](#booting-machines-with-rhcos-and-ignition-configs)
+
+6. [Watching your installation (bootstrap complete, cluster available)](#watching-your-installation)
+
+7. [Example Bare-Metal UPI deployment](#example-power-upi-configuration)
+
+## Compute
+
+The smallest OpenShift 4.x clusters require the following host:
+
+* 1 bootstrap machine.
+
+* 3 control plane machines.
+
+* atleast 1 worker machine.
+
+NOTE: The cluster requires the bootstrap machine to deploy the OpenShift cluster on to the 3 control plane machines, and you can remove the bootstrap machine.
+
+All the machines must use Red Hat Enterprise Linux CoreOS (RHCOS) as the operating system.
+
+### Minimum resource requirements
+
+| Machine       | Operating System | vCPU | RAM   | Storage |
+|---------------|------------------|------|-------|---------|
+| Bootstrap     | RHCOS            | 2    | 16 GB | 120 GB  |
+| Control Plane | RHCOS            | 2    | 16 GB | 120 GB  |
+| Compute       | RHCOS            | 2    | 16 GB | 120 GB  |
+
+
+
+## Network Topology Requirements
+
+The easiest way to get started is to ensure all Power nodes have internet access to pull images for platform containers and provide telemetry data to Red Hat.
+OpenShift 4.x also supports air-gapped installs
```

Comment:
nit: doc use "restricted network", because those sorts of issues happen in many situations, not all of which are actually air-gapped.  And do we need to call this out at all?  Saying "connected is easiest, but restricted-network is possible" without going into the details seems like an unnecessary straddle.  Can we either drop this, or explain the restricted-network steps?

Reply:
@wking the disconnected install doc is currently WIP. I'll add a link to it in a follow-on PR.